scout/server/blueprints/institutes/controllers.py

```diff
@@ -598,13 +603,37 @@ def get_and_set_cases_by_status(
             case_groups[status].append(case_obj)
             nr_cases_showall_statuses += 1
 
+    def get_specific_query(request):
+        """Check if only (non-)specific query terms were used in query.
+        If so we assume this is a default query, and dim all cases
+        that match the "show_all_cases_status", highlighting the
+        (max limit number) other cases that were returned.
+        """
+        cases_query = store.cases(
+            collaborator=institute_obj["_id"],
+            name_query=request.form,
+            skip_assigned=request.form.get("skip_assigned"),
+            is_research=request.form.get("is_research"),
+            has_rna_data=request.form.get("has_rna"),
+            verification_pending=request.form.get("validation_ordered"),
+            has_clinvar_submission=request.form.get("clinvar_submitted"),
+            yield_query=True,
+        )
+        for key, value in cases_query.items():
```

Comment:
What is happening here, is this checking that cases key are not in ["collaborators", " ", ""]. Why?